src/main/scala/zio/zmx/prometheus/PrometheusDataModel.scala

```diff
@@ -0,0 +1,136 @@
+package zio.zmx.prometheus
+import zio.Chunk
+
+trait Metric {
+  def name: String
+  def help: Option[String]
+  def labels: Map[String, String]
+}
+
+object Metric {
+  sealed trait BucketType {
+    def buckets: List[Double]
+  }
+  object BucketType       {
+
+    type Buckets = Map[Double, Counter]
```

Comment:
Idea: Using a case class here instead of a type could help to build operation between Buckets (merge, ...)